Kristen-Hu/Kristen-Hu.md

```diff
@@ -0,0 +1,14 @@
+---
+id: "05daa2a9-8768-446e-b9d3-580c8a6f9b8b"
+
+category: "Developers"
+role: "Sample Profile"
```

Comment:
Can you replace "Sample Profile" with your job title

Reply:
Done.